lib/cocoapods-core/specification/root_attribute_accessors.rb

```diff
@@ -152,6 +152,14 @@ def prepare_command
           command.strip_heredoc.chomp if command
         end
 
+        # @return [Hash] Return the outermost spec
+        #
+        def ancestor
```

Comment:
This shouldn't be necessary, there's already a `Specification#root` method that returns the root specification